### PR TITLE
fix some spelling

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Http.Abstractions/Internal/HeaderSegmentCollection.cs [hunk 1]
```diff
@@ -167,7 +167,7 @@ public bool MoveNext()
                         // if that was the last string
                         if (_index == _headers.Count)
                         {
-                            // no more move nexts
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Http.Abstractions/Internal/HeaderSegmentCollection.cs]: I think this one was intentional.

File: src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs [hunk 2]
```diff
@@ -123,7 +123,7 @@ public override bool Equals(object obj)
                 return false;
             }
 
-            // RFC2616: 14.20: unquoted tokens should use case-INsensitive comparison; quoted-strings should use
+            // RFC2616: 14.20: unquoted tokens should use case-Insensitive comparison; quoted-strings should use
```
Comment (JamesNK) [inline src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs]: This one looks intentional

File: src/Microsoft.AspNetCore.Owin/WebSockets/OwinWebSocketAcceptAdapter.cs [hunk 3]
```diff
@@ -103,10 +103,10 @@ private void EnsureCompleted(Task task)
         // 2. The middleware inserts an alternate Accept signature into the OWIN environment.
         // 3. The middleware invokes Next and stores Next's Task locally. It then returns an alternate Task to the server.
         // 4. The OwinFeatureCollection adapts the alternate Accept signature to IHttpWebSocketFeature.AcceptAsync.
-        // 5. A component later in the pipleline invokes IHttpWebSocketFeature.AcceptAsync (mapped to AcceptWebSocketAsync).
+        // 5. A component later in the pipeline invokes IHttpWebSocketFeature.AcceptAsync (mapped to AcceptWebSocketAsync).
         // 6. The middleware calls the OWIN Accept, providing a local callback, and returns an incomplete Task.
         // 7. The middleware completes the alternate Task it returned from Invoke, telling the server that the request pipeline has completed.
-        // 8. The server invokes the middleware's callback, which creats a WebSocket adapter complete's the orriginal Accept Task with it.
+        // 8. The server invokes the middleware's callback, which creates a WebSocket adapter completes the original Accept Task with it.
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Owin/WebSockets/OwinWebSocketAcceptAdapter.cs]: adapter and completes

**Comment by JamesNK** [comment]
Very nice. What tool did you use for spell checking?

File: src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs (hunk 2 above)
Comment (SimonCropp) [reply src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs]: looking at the spec, it always uses `case-insensitive`. so not sure why the comment would diverge from that?

File: src/Microsoft.AspNetCore.Http.Abstractions/Internal/HeaderSegmentCollection.cs (hunk 1 above)
Comment (SimonCropp) [reply src/Microsoft.AspNetCore.Http.Abstractions/Internal/HeaderSegmentCollection.cs]: ok. reverted

File: src/Microsoft.AspNetCore.Owin/WebSockets/OwinWebSocketAcceptAdapter.cs (hunk 3 above)
Comment (SimonCropp) [reply src/Microsoft.AspNetCore.Owin/WebSockets/OwinWebSocketAcceptAdapter.cs]: yep. fixed

**Comment by SimonCropp** [comment]
@JamesNK r#

**Comment by SimonCropp** [comment]
what are tspecials?
```
// tspecials are required to be in a quoted string.  Only non-ascii needs to be encoded.
```

File: src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs (hunk 2 above)
Comment (JamesNK) [reply src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs]: emphasis

**Comment by JamesNK** [comment]
@Tratcher should this be rebased on release/2.2 branch?

**Comment by Tratcher** [comment]
Having this in release/2.2 would be nice but is not blocking. @SimonCropp want to rebase?

**Comment by SimonCropp** [comment]
cherry picked+squashed+resubmitted https://github.com/aspnet/HttpAbstractions/pull/1035

**Pull request closed without merging**